spec/system/candidate_interface/candidate_account_spec.rb

```diff
@@ -36,6 +38,10 @@
     then_i_should_be_signed_out
   end
 
+  def given_the_pilot_is_open
+    FeatureFlag.activate('pilot_open')
```

Comment:
Do we need to tear this down afterwards?

Reply:
Perhaps, but I thought it added overhead and all of the tests should do the correct setup themselves.